camel/storages/graph_storages/neo4j_graph.py

```diff
@@ -0,0 +1,548 @@
+                "'apoc.meta.data()' is allowed in Neo4j configuration ")
+
+    @property
+    def client(self) -> Any:
+        return self.driver
+
+    @property
+    def get_schema(self, refresh: bool = False) -> str:
+        r"""Retrieve the schema of the Neo4jGraph store.
+
+        Args:
+            refresh (bool): A flag indicating whether to forcibly refresh the
+                schema from the Neo4jGraph store regardless of whether it is
+                already cached. Defaults to `False`.
+
+        Returns:
+            str: The schema of the Neo4jGraph store.
+        """
+        if self.schema and not refresh:
+            return self.schema
+        self.refresh_schema()
+        logger.debug(f"get_schema() schema:\n{self.schema}")
+        return self.schema
+
+    @property
+    def get_structured_schema(self) -> Dict[str, Any]:
+        r"""Returns the structured schema of the graph
+
+        Returns:
+            dict[str, any]: The structured schema of the graph.
+        """
+        return self.structured_schema
+
+    def _value_sanitize(self, d: Any) -> Any:
+        r"""Sanitizes the input dictionary `d` by removing entries that are
+        dictionaries with values resembling embeddings and lists containing
+        more than `LIST_LIMIT` elements. This method aims to reduce unnecessary
+        computational cost and noise in scenarios where such detailed data
+        structures are not needed, for example, in simplifying data for
+        processing by a language model.
+
+        Args:
+            d (Any): The dictionary to be sanitized.
+
+        Returns:
+            Any: The sanitized dictionary, with embedding-like
+                dictionaries and oversized lists handled.
+        """
+        if isinstance(d, dict):
+            new_dict = {}
+            for key, value in d.items():
+                if isinstance(value, dict):
+                    sanitized_value = self._value_sanitize(value)
+                    # Check if the sanitized value is not None
+                    if (sanitized_value is not None):
+                        new_dict[key] = sanitized_value
+                elif isinstance(value, list):
+                    if len(value) < LIST_LIMIT:
+                        sanitized_value = self._value_sanitize(value)
+                        # Check if the sanitized value is not None
+                        if (sanitized_value is not None):
+                            new_dict[key] = sanitized_value
+                    # Do not include the key if the list is oversized
+                else:
+                    new_dict[key] = value
+            return new_dict
+        elif isinstance(d, list):
+            if len(d) < LIST_LIMIT:
+                return [
+                    self._value_sanitize(item) for item in d
+                    if self._value_sanitize(item) is not None
+                ]
+            else:
+                return None
+        else:
+            return d
+
```

Comment:
Why not throw an error or warning? If the func returns `d` without any process, it seems not very seasonable.

Reply:
there may have other possible types like string